objects_delete/delete_temp_objects_job.py

```diff
@@ -0,0 +1,33 @@
+from time import time
+from gen3 import object, metadata, auth
+
+auth = auth.Gen3Auth(refresh_file="/mnt/api-profile-credentials.json")
+mds_handle = metadata.Gen3Metadata(auth)
+
+print("Initializing delete_temp_objects_job...")
+LIMIT_SIZE = 2
+offset_position = 0
+response_dict = mds_handle.query(
+    query="date_to_delete=*", return_full_metadata=True, limit=LIMIT_SIZE
+)
+if type(response_dict) is dict:
+    guid_list = list(response_dict.values())
+    while len(response_dict) == LIMIT_SIZE:
+        offset_position += LIMIT_SIZE
+        response_dict = mds_handle.query(
+            query="date_to_delete=*",
+            return_full_metadata=True,
+            limit=LIMIT_SIZE,
+            offset=offset_position,
+        )
+        if response_dict:
```

Comment:
why this check?

Reply:
Aah! I guess I wouldn't need it now. Previously I was building a list of lists, where I was appending all the smaller lists into a big one and then iterating them. That is when I placed this check to avoid adding `[]`. But on careful examination, I realized I was unnecessarily over working the logic and changed it to a single list. This check still stayed. I can take it off. Not a problem. :D